Composite/Plugins/Functions/FunctionProviders/StandardFunctionProvider/Pages/GetPageIdFunction.cs

```diff
@@ -70,30 +71,45 @@ private Guid GetCurrentPageId()
         private Guid? GetCurrentPageIdFromHttpContext()
         {
             var entityToken = HttpContext.Current?.Items[ActionExecutorFacade.HttpContextItem_EntityToken];
-            return GetPageIdFromDataToken(entityToken as DataEntityToken);
+            return GetPageIdFromDataToken(entityToken);
         }
 
         private Guid? GetCurrentPageIdFromFormFlow()
         {
             var currentFormTreeCompiler = FormFlowUiDefinitionRenderer.CurrentFormTreeCompiler;
             if (currentFormTreeCompiler.BindingObjects.TryGetValue(FormsWorkflow.EntityTokenKey, out var entityToken))
             {
-                return GetPageIdFromDataToken(entityToken as DataEntityToken);
+                return GetPageIdFromDataToken(entityToken);
             }
 
             return null;
         }
 
-        private Guid? GetPageIdFromDataToken(DataEntityToken entityToken)
+        private Guid? GetPageIdFromDataToken(object entityTokenObj)

```

Comment:
If we get pageId from EntityToken we should rename to GetPageIdFromEnitityToken

Reply:
Right, thanks:) Renamed